model/src/main/kotlin/config/PackageCurationProviderConfiguration.kt

```diff
@@ -30,6 +30,11 @@ data class PackageCurationProviderConfiguration(
     @ConfigAlias("name")
     val type: String,
 
+    /**
+     * A unique identifier for the package curation provider.
+     */
+    val id: String,
```

Comment:
I would prefer if this would default to `type` to not require specifying an id as long as no two providers of the same type are used.